app/dashboard/templates/profile_details.html

```diff
@@ -230,19 +188,22 @@ <h6>
         </div>
 
 
-    {% include 'shared/bottom_notification.html' %}
-    {% include 'shared/analytics.html' %}
-    {% include 'shared/footer_scripts.html' %}
-    {% include 'shared/rollbar.html' %}
-    {% include 'shared/footer.html' %}
+        {% include 'shared/bottom_notification.html' %} {% include 'shared/analytics.html' %} {% include 'shared/footer_scripts.html'%}
+        {% include 'shared/rollbar.html' %} {% include 'shared/footer.html' %}
```

Comment:
Stick to 2 spaces 
And let's keep each of them in their own line ^_^
Easier to read them that way